README.md

```diff
@@ -178,20 +190,55 @@ The Web view is utilized to:
 > * prevents Wallets / Mobile Apps from having to update source code in order to integrate with different PFIs
 > * establish a clear distinction between the application initiating the flow and a PFI
 
+## Multiple Credential Types
+
+Financial service providers often perform KYC incrementally in response to customer's expressed intent to access services. For example, minimal upfront diligence on newly onboarded customers, followed by additional diligence when the customer's activity passes a threshold.
 
-## Initiating IDV Flow
+A KCC Issuer may define multiple KCC types to represent different levels of KYC diligence, and describe each with a Presentation Definition. Mobile Apps who hold a KCC already may apply for new credentials to acquire a higher level KCC (see [Initiate Application](#request)) 
 
-Initiating the IDV flow is done using [SIOPv2](https://openid.github.io/SIOPv2/openid-connect-self-issued-v2-wg-draft.html).
+# Credential Application Flow
+
+```mermaid
+sequenceDiagram
+
+actor A as Applicant
+participant W as Webview
+participant D as Mobile App
+participant P as PFI
+participant I as IDV
+
+rect rgba(0, 0, 0, 0.1)
+    D->>+P: Initiate KCC application
+    Note right of I: Initiate Application
+    P-->>-D: IDV Request
```

Comment:
Nice love this swimlane

But why is this the only `-->>` (dotted) line?

Reply:
Using dotted lines for HTTP responses. I think the Issue KCC line should be dotted too